lib/haml_lint.rb

```diff
@@ -3,35 +3,35 @@
 # Need to load haml before we can reference some Haml modules in our code
 require 'haml'
 
-require 'haml_lint/constants'
-require 'haml_lint/exceptions'
-require 'haml_lint/configuration'
-require 'haml_lint/configuration_loader'
-require 'haml_lint/document'
-require 'haml_lint/haml_visitor'
-require 'haml_lint/lint'
-require 'haml_lint/linter_registry'
-require 'haml_lint/ruby_parser'
-require 'haml_lint/linter'
-require 'haml_lint/logger'
-require 'haml_lint/reporter'
-require 'haml_lint/report'
-require 'haml_lint/linter_selector'
-require 'haml_lint/file_finder'
-require 'haml_lint/runner'
-require 'haml_lint/utils'
-require 'haml_lint/version'
-require 'haml_lint/version_comparer'
-require 'haml_lint/severity'
+require_relative 'haml_lint/constants'
```

Comment:
Why would this not be ./constants? (Just curious)

Reply:
Hi,

no because `haml_lint` is in `lib`, and constants is in `lib/haml_lint/constants.rb`, so the relative path does include `haml_lint`

```
lib/
├── haml_lint/
│   └── constants.rb
└── haml_lint.rb
```

This is a good reference in a (former) ruby core gem: https://github.com/ruby/logger/blob/d1d704a6336a82903011336a0116012e9aa4a504/lib/logger.rb#L17-L21

Reply:
Some other reference in Ruby core that are not linked in this PR:

- https://github.com/ruby/psych/pull/522
- https://github.com/ruby/rdoc/pull/658

Reply:
Reading from the GitHub app on my phone was throwing me off. Pathing makes sense 👍